src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -22,16 +21,11 @@ export const AccountDetailsPage: FC = () => {
   const address = useLoaderData() as string
   const accountQuery = useGetRuntimeAccountsAddress(network, layer, address)
   const account = accountQuery.data?.data
-  const rosePriceQuery = useGetRosePrice()
```

Comment:
hm, keeping requests in Page components is becoming a bit annoying. I would still stick to it tho. Makes tests, storybooks, and reusing components clearer

Reply:
This comment actually belongs to https://github.com/oasisprotocol/explorer/pull/463, where this change was made. (This PR is build on top of that PR.)